packages/common/src/services/audius-api-client/types.ts

```diff
@@ -26,6 +26,7 @@ export type APIUser = {
   artist_pick_track_id: Nullable<OpaqueID>
   blocknumber: number
   balance: string
+  usdc_balance: string
```

Comment:
IIRC we decided that we wouldn't return `usdc_balance` from the API and just fetch it from chain

Reply:
Going to remove it for now. We can add it back if needed when implementing the balance tile.